cmd/edit/cluster/cmd.go

```diff
@@ -966,3 +962,13 @@ func auditLogInteractivePrompt(r *rosa.Runtime, cmd *cobra.Command, cluster *cmv
 	}
 	return
 }
+
+func PromptUserToAcceptRegistryChange(r *rosa.Runtime) bool {
+	prompt := "Changing any registry related parameter will trigger a rollout across all machinepools " +
+		"(all machinepool nodes will be recreated, following pod draining from each node). Do you want to proceed?"
+	if !confirm.ConfirmRaw(prompt) {
+		r.Reporter.Warnf("No changes to registry configuration.")
+		return false
+	}
+	return true
+}
```

Comment:
Codecov isn't linking this because we have so many untested lines from this function gating the code above.

I think a solution, would be to move the logic inside the `if` statement above (L662 -> L676) to its own function, and test it if possible.

Another solution would be to get codecov to ignore this, but I'm not sure if that's possible cc: @robpblake 

LGTM to me though 😄

Reply:
specifically, moving this code to a func which is separate from input logic;

```clusterConfig.AllowedRegistries = allowedRegistries
 				clusterConfig.BlockedRegistries = blockedRegistries
 				clusterConfig.InsecureRegistries = insecureRegistries
 				clusterConfig.PlatformAllowlist = platformAllowlist
 				if additionalTrustedCa != "" {
 					ca, err := clusterregistryconfig.BuildAdditionalTrustedCAFromInputFile(additionalTrustedCa)
 					if err != nil {
 						r.Reporter.Errorf("Failed to build the additional trusted ca from file %s, got error: %s",
 							additionalTrustedCa, err)
 						os.Exit(1)
 					}
 					clusterConfig.AdditionalTrustedCa = ca
 					clusterConfig.AdditionalTrustedCaFile = additionalTrustedCa
 				}
 				clusterConfig.AllowedRegistriesForImport = allowedRegistriesForImport```

Reply:
ACTUALLY, talked to rob and he said code cov is in informative mode;

```It looks like creators_test.go is failing:

[FAILED] Expected
      <string>: "..._hcp_suppor..."
  to equal               |
      <string>: "..._hcp_instal..."
  In [It] at: /go/src/github.com/openshift/rosa/cmd/create/accountroles/creators_test.go:30 @ 10/17/24 15:34:04.183```